deps/build.jl

```diff
@@ -7,6 +7,8 @@ using BinDeps, Compat
     provides(AptGet, "hdf5-tools", hdf5)
     provides(Pacman, "hdf5", hdf5)
     provides(Yum, "hdf5", hdf5)
+    provides(Sources, URI("http://www.hdfgroup.org/ftp/HDF5/current/src/hdf5-1.8.16.tar.gz"), hdf5)
+    provides(BuildProcess, Autotools(libtarget = "libhdf5.la"), hdf5)
```

Comment:
it would make sense on os x to have a source-build backup option in case something goes wrong with homebrew.jl